src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,17 @@
 package racingcar;
 
+import static racingcar.OutputView.printCars;
+import static racingcar.OutputView.printWinners;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        String carName = InputView.readCarNamesInput();
+        int raceTime = InputView.readRaceTimeInput();
+
+        Cars cars = new Cars(carName);
+
+        printCars(raceTime, cars);
+        printWinners(cars);
```

Comment:
여기도 `InputView.readCarNamesInput();` 처럼
```suggestion
        OutputView.printCars(raceTime, cars);
        OutputView.printWinners(cars);
```
이렇게 호출하면 일관성 있어서 가독성이 더 좋아질것 같아요!
또는 InputView 도 import해서 메소드를 바로 호출하던가 해서 일관성이 중요할것 같습니다!

Reply:
아하 맞는 것 같습니다. 이게 맞네요. 정적으로 하던가 인스턴스로 하던가 둘 중 하나만 했어야 되는 것 같습니다